app/helpers/organisation_helper.rb

```diff
@@ -2,4 +2,31 @@ module OrganisationHelper
   def child_organisations_count(organisation)
```

Comment:
I know this isn't part of this PR, but it has a very similar name to other `count` methods.  Maybe consider whether changing the name of it would help understand which one you should use?  What's the difference between `works_with` and `ordered_child_organisations`?